src/styles/components/Hero.scss

```diff
@@ -1,79 +1,65 @@
-.Hero {
-  min-height: 92vh;
-
-  &__nav-headline {
-    text-align: right;
-  }
-
-  &__link:hover {
-    color: color('white');
-    text-decoration: underline;
-  }
+$iframe-height-mobile: 28rem;
```

Comment:
might be a good idea to store all the variables in one file?

Reply:
@mokaymokay fixed it! :)